### PR TITLE
Added team member

### PR DESCRIPTION
**Pull request by mrleewong**

### CODE REVIEW

File: _data/team-management.yml [hunk 1]
```diff
@@ -86,6 +86,9 @@
   image: anon.png
```
Comment (nmarley) [inline _data/team-management.yml]: Can we remove the 3 empty fields here?

File: _data/team-management.yml [hunk 2]
```diff
@@ -80,3 +80,17 @@
     - Management
     - Finance
 
+- name: Bob Carroll
+  title: CTO
+  description:
+  image: anon.png
+  email:
```
Comment (nmarley) [inline _data/team-management.yml]: Can we remove empty email, anon image and empty keybase here?

File: _data/team-management.yml [hunk 3]
```diff
@@ -80,3 +80,17 @@
...
+  keybase:
+  image: bob_carroll.jpg
+  email: bob@dash.org
+  keybase: https://keybase.io/bobcarroll
+  twitter:
+  dashforum-link: /bob.27367/
+  dashforum: bob
+  team-title:
+    - Management
```
Comment (nmarley) [inline _data/team-management.yml]: Can you also add "Development" team here?

Comment (mrleewong) [reply _data/team-management.yml]: @nmarley On the next line

**Pull request merged**